docs/getting-started/powerstrip/CalicoSwarm.md

```diff
@@ -89,7 +89,7 @@ export ETCD_AUTHORITY=$MANAGER_IP:4001
 ```
 
 ## Installing Swarm on your cluster
-Now that Calico networking is configured on our cluster, lets join our nodes into a Docker Swarm cluster.
+Now that Calico networking is configured on our cluster, lets join our nodes into a [Docker Swarm][swarm] cluster.
```

Comment:
Not too fussed if you change this since we do this elsewhere, but I'd generally not hyperlink every occurrence of things like this.  We hyperlinked it at the top, and if they got this far then they know what Swarm is.